cds_ils/migrator/eitems/api.py

```diff
@@ -236,12 +241,17 @@ def migrate_ezproxy_links(raise_exceptions=True):
         Document = current_app_ils.document_record_cls
         document = Document.get_record_by_pid(hit.pid)
         click.echo("Processing document {}...".format(document["pid"]))
-
-        for url in document["_migration"]["eitems_proxy"]:
+        open_access = document["_migration"]["eitems_open_access"]
```

Comment:
why did you add open_access depending on this field when login_required=True ?

Reply:
I am not sure I understand the question. Do you mean that if login_required=True then we should not take into account open_access?

Reply:
could you check with @ntarocco about how login_required influences the open_access? maybe we have it documented somewhere, he worked quite heavily on that part

Reply:
Here more details: https://github.com/CERNDocumentServer/cds-ils/issues/162

Reply:
It seems that there is no relation between login_required and open_access fields, so we respect marcXML fields values.